src/store/server/fetchQueries.js

```diff
@@ -0,0 +1,18 @@
+import 'rxjs/add/operator/map';
```

Comment:
this is a new module that just encapsulates the server-side equivalent of `fetchQueries` - the code was moved out of `src/store/server/index.js` just to keep the structure parallel with `src/store/browser/index.js`.